[![DOI](https://zenodo.org/badge/143102045.svg)](https://zenodo.org/badge/latestdoi/143102045)


# APPLS
Applied Python Programming for Life Scientists

This course is intended for life scientists without any prior knowledge. The objective is to provide a sufficient amount of knowledge about Python to solve small problems by writing simple scripts. Feel free to re-use my slides and please report any mistakes that you discover. I am planning to provide an updated version in the future.
Please read the wiki for further information.



The last part contains examples from my published research:

Pucker, B., Holtgräwe, D., Rosleff Sörensen, T., Stracke, R., Viehöver, P., and Weisshaar, B. (2016). A de novo Genome Sequence Assembly of the Arabidopsis thaliana Accession Niederzenz-1 Displays Presence/Absence Variation and Strong Synteny. PloS‑ONE 11:e0164321.

[full text](https://doi.org/10.1371/journal.pone.0164321)



Pucker, B., Holtgräwe, D., & Weisshaar, B. (2017). Consideration of non-canonical splice sites improves gene prediction on the Arabidopsis thaliana Niederzenz-1 genome sequence. BMC Research Notes, 10, 667. doi:10.1186/s13104-017-2985-y

[full text](https://doi.org/10.1186/s13104-017-2985-y)



Haak, M., Vinke, S., Keller, W., Droste, J., Rückert, C., Kalinowski, J., & Pucker, B. (2018). High Quality de Novo Transcriptome Assembly of Croton tiglium. Frontiers in Molecular Biosciences, 5. doi:10.3389/fmolb.2018.00062


[full text](https://doi.org/10.3389/fmolb.2018.00062)



Additional courses are listed here:

https://www.cebitec.uni-bielefeld.de/~bpucker
